nbstripout.py

```diff
@@ -90,6 +90,14 @@
 from argparse import ArgumentParser, RawDescriptionHelpFormatter
 import io
 import sys
+import os
+
+# sys.stdin is None sometimes, make sure std* streams are well defined
+# from https://stackoverflow.com/a/17459258
+for _name in ('stdin', 'stdout', 'stderr'):
```

Comment:
As discussed in #80 the other option is deferring creation of `input_stream` until we know it'll be used i.e. `args.files` is empty. I'm happy with either version.

Does the TortoiseGit GUI also close `stdout` and `stderr`? If yes then your current solution is preferable I think.